src/theme/JSONSchemaViewer/labels/Constant/styles.module.css

```diff
@@ -0,0 +1,4 @@
+.constant {
+    font-size: var(--ifm-code-font-size);
+    color: var(--ifm-color-secondary-darkest);
+}
```

Comment:
Note no new line at end of file

Reply:
Should not be an issue for now ;)

Reply:
remind me not to `cat` your css files ;)